watertap/unit_models/tests/test_anaerobic_digestor.py

```diff
@@ -73,44 +54,9 @@
 
 
 # -----------------------------------------------------------------------------
-@pytest.mark.unit
-def test_config():
-    m = ConcreteModel()
-
-    m.fs = FlowsheetBlock(dynamic=False)
-
-    m.fs.props = ADM1ParameterBlock()
-    m.fs.props_vap = ADM1_vaporParameterBlock()
-    m.fs.rxn_props = ADM1ReactionParameterBlock(property_package=m.fs.props)
-
-    m.fs.unit = AD(
-        liquid_property_package=m.fs.props,
-        vapor_property_package=m.fs.props_vap,
-        reaction_package=m.fs.rxn_props,
-    )
-
-    assert len(m.fs.unit.config) == 16
-
-    assert not m.fs.unit.config.dynamic
-    assert not m.fs.unit.config.has_holdup
-    assert m.fs.unit.config.material_balance_type == MaterialBalanceType.useDefault
-    assert m.fs.unit.config.energy_balance_type == EnergyBalanceType.useDefault
-    assert m.fs.unit.config.momentum_balance_type == MomentumBalanceType.pressureTotal
-    assert not m.fs.unit.config.has_heat_transfer
-    assert not m.fs.unit.config.has_pressure_change
-    assert not m.fs.unit.config.has_equilibrium_reactions
-    assert not m.fs.unit.config.has_phase_equilibrium
-    assert not m.fs.unit.config.has_heat_of_reaction
-    assert not m.fs.unit.config.has_pressure_change
-    assert m.fs.unit.config.liquid_property_package is m.fs.props
-    assert m.fs.unit.config.vapor_property_package is m.fs.props_vap
-    assert m.fs.unit.config.reaction_package is m.fs.rxn_props
-
-
-# -----------------------------------------------------------------------------
-class TestAdm(object):
-    @pytest.fixture(scope="class")
-    def adm(self):
+class TestUnitDefault(UnitTestHarness):
+    @pytest.mark.unit
+    def configure(self):
```

Comment:
```suggestion

    def configure(self):
```

I'm not sure the marker here on a non-`test_` function would have any effect.